File: Examples/VisualBasic/ProgrammersGuide/WorkingWithProjects/WriteFiscalYearProperties.vb
'////////////////////////////////////////////////////////////////////////
' Copyright 2001-2014 Aspose Pty Ltd. All Rights Reserved.
'
' This file is part of Aspose.Tasks. The source code in this file
' is only intended as a supplement to the documentation, and is provided
' "as is", without warranty of any kind, either expressed or implied.
'////////////////////////////////////////////////////////////////////////

Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Tasks
Imports Aspose.Tasks.Saving

Namespace VisualBasic.Projects
	Public Class WriteFiscalYearProperties
		Public Shared Sub Run()
			' The path to the documents directory.
			Dim dataDir As String = RunExamples.GetDataDir_Projects()

            ''Create a project instance
            'Dim prj As New Project(dataDir & "WriteFiscalYearProperties.mpp")
            ''Set fiscal year properties
            '         prj.[Set](prj.FyStartDate, Month.July)
            '         prj.[Set](prj.FiscalYearStart, True)
            '         prj.Save(dataDir & "saved.mpp", SaveFileFormat.MPP)
		End Sub
	End Class
End Namespace

File: Examples/CSharp/HandlingBasicProperties/WriteFiscalYearProperties.cs
using System;
using Aspose.Tasks.Saving;

namespace Aspose.Tasks.Examples.CSharp.HandlingBasicProperties
{
    public class WriteFiscalYearProperties
    {
        public static void Run()
        {
            // ExStart:WriteFiscalYearProperties

            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_HandlingBasicProperties();

            try
            {
                // Create a project instance
                Project prj = new Project(dataDir + "WriteFiscalYearProperties.mpp");

                // Set fiscal year properties
                prj.Set(Prj.FyStartDate, Month.July);
                prj.Set(Prj.FiscalYearStart, true);
                prj.Save(dataDir + "saved.mpp", SaveFileFormat.MPP);
            }
            catch (Exception ex)
            {
                Console.WriteLine(ex.Message + "\nThis example will only work if you apply a valid Aspose License. You can purchase full license or get 30 day temporary license from http:// Www.aspose.com/purchase/default.aspx.");
            }
            // ExEnd:WriteFiscalYearProperties
        }
    }
}